tests/framework/e2e/downgrade.go

```diff
@@ -143,7 +143,7 @@ func ValidateMemberVersions(t *testing.T, epc *EtcdProcessCluster, expect []*ver
 }
 
 func ValidateVersion(t *testing.T, cfg *EtcdProcessClusterConfig, member EtcdProcess, expect version.Versions) {
-	testutils.ExecuteWithTimeout(t, 1*time.Minute, func() {
+	testutils.ExecuteWithTimeout(t, 2*time.Minute, func() {
```

Comment:
same question why it takes so long?

Reply:
ditto (https://github.com/etcd-io/etcd/pull/19366#discussion_r1949843196)